docs/setup.rst

```diff
@@ -0,0 +1,28 @@
+Account setup
```

Comment:
Account creation?

Also change the name of file from setup to the title of this section in snake case.